android/app/src/main/java/io/mosip/registration_client/MainActivity.java

```diff
@@ -138,11 +144,24 @@ public void configureFlutterEngine(@NonNull FlutterEngine flutterEngine) {
     }
 
     public void getUISchema(MethodChannel.Result result){
+        try{
+
+            String schemaJson = identitySchemaRepository.getSchemaJson(getApplicationContext(),
+                    identitySchemaRepository.getLatestSchemaVersion());
+            result.success(schemaJson.toString());
+
+        }catch (Exception e){
+
+        }
+    }
+    public void getNewProcessSpec(MethodChannel.Result result){
         try{
 
             ProcessSpecDto processSpecDto = identitySchemaRepository.getNewProcessSpec(getApplicationContext(),
                     identitySchemaRepository.getLatestSchemaVersion());
-            result.success(processSpecDto.toString());
+            ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
+            String json = ow.writeValueAsString(processSpecDto);
+            result.success(json);
 
         }catch (Exception e){
 
```

Comment:
Kindly add logger.error

Reply:
In  the samefile, line number: 175, why are we returning empty center name on exception?

Reply:
Done and Fixed.